src/main/java/uk/sky/cqlmigrate/CqlFileParser.java

```diff
@@ -22,37 +22,139 @@ private CqlFileParser() {}
 
     private static final char CQL_STATEMENT_STRING_DELIMITER = '\'';
     private static final String CQL_STATEMENT_TERMINATOR = ";";
+    private static final String CQL_COMMENT_DOUBLE_HYPEN = "--"; //Double hypen
+    private static final String CQL_MULTI_LINE_COMMENT_OPEN = "/*"; //Forward slash asterisk
+    private static final String CQL_MULTI_LINE_COMMENT_CLOSE = "*/"; //Asterisk forward slash
+    private static final Pattern CQL_MULTI_LINE_COMMENT_PATTERN = Pattern.compile("/\\*.*?\\*/", Pattern.DOTALL);
+    private static final Pattern EOL = Pattern.compile(".*\\R|.+\\z");
+    private static final String EMPTY_STR = "";
 
     static List<String> getCqlStatementsFrom(Path cqlPath) {
-        String cqlFileAsString;
+        final LineProcessor processor = new LineProcessor();
```

Comment:
Unfortunately we don't have any code style guidelines, but normally we wouldn't use the `final` keyword for variables.